platform/darwin/src/MGLStyle.mm

```diff
@@ -254,5 +257,21 @@ - (void)removeStyleClass:(NSString *)styleClass
     }
 }
 
+- (void)setImage:(MGLImage *)image forName:(NSString *)name
+{
+    if (image && name)
```

Comment:
Assert instead of failing gracefully. These two parameters are implied to be `nonnull` in the header.